twilight-gateway/src/compression/inflater.rs

```diff
@@ -142,33 +139,26 @@ impl Inflater {
             // It doesn't matter if we lose precision for logging.
             #[allow(clippy::cast_precision_loss)]
             let saved_percentage =
-                self.decompress.total_in() as f64 / self.decompress.total_out() as f64;
-            let saved_percentage_readable = saved_percentage * 100.0;
+                self.decompress.total_in() as f64 * 100.0 / self.decompress.total_out() as f64;
             let saved_kib = (self.decompress.total_out() - self.decompress.total_in()) / 1_024;
 
             tracing::trace!(
-                saved_kib = saved_kib,
-                saved_percentage = %saved_percentage_readable,
-                shard_id = %self.shard_id,
-                total_in = self.decompress.total_in(),
-                total_out = self.decompress.total_out(),
-                "data saved",
+                "{} KiB saved in total ({:.2}%)",
```

Comment:
This is statistical information, not debugging information, and is also logged in `inflater_metrics`. Should it be removed entirely?

Reply:
I think we can keep it for now, but we should probably add another way to get metrics from the inflater without using the metrics crate.